pom.xml

```diff
@@ -4,7 +4,7 @@
 
     <groupId>org.openjump</groupId>
     <artifactId>topology-extension</artifactId>
-    <version>2.0.3</version>
+    <version>2.0.4-SNAPSHOT</version>
```

Comment:
please remove "-SNAPSHOT" here. this is added automagically by maven when snapshots are built.

Reply:
nevermind, just realized we are not in OJ-core. my bad :), sorry for the noise